papa/src/main/java/papa/internal/RealInputTracker.kt

```diff
@@ -160,9 +167,17 @@ internal object RealInputTracker : InputTracker {
         window.keyEventInterceptors += KeyEventInterceptor { keyEvent, dispatch ->
           val traceSectionName = keyEvent.traceSectionName
           val now = System.nanoTime()
+          val deliveryUptimeMillis = now.nanoseconds.inWholeMilliseconds
           val cookie = now.rem(Int.MAX_VALUE).toInt()
           SafeTrace.beginAsyncSection(traceSectionName, cookie)
-          val input = DeliveredInput(keyEvent, now.nanoseconds, 0) {
+          // if eventTime is after delivery uptime, use the eventUptime value for DeliveredInput.
```

Comment:
typo: "use the eventUptime" => "use the deliveryUptime"

Reply:
👍